### DISCUSSION TITLE
Limiter: fake driver for testing so you don't have to setup redis/db

### DISCUSSION THREAD

**Discussion by maximehuylebroeck** [adonisjs/core#3833] [Ideas]
Hey,
In recent light of better testing tools in adonisjs.
I was thinking damn wouldn't it be nice to have a fake in memory driver for limiter so you don't have the pain of setting up a local redis/db.
In a nutshell it would do exactly the same as the limiter drivers right now but will user the memory limiter under the hood.

Let me know what you think about this.

edit: already created a PR for this

**Comment by HasanAshab** [comment]
Adonis docs said there is a Memory driver. However when i use 'memory' driver, it gives error
